### PR TITLE
Fix Session Page Of Mentor Dashboard

### PR DESCRIPTION
**Pull request by coder12git**
I have designed the session page.
Screenshot 2023-08-06 221403
Screenshot 2023-08-06 221341

### CODE REVIEW

**Comment by coder12git** [comment]
Screenshot 2023-08-06 234624
@anmode

**Comment by coder12git** [comment]
Screenshot 2023-08-07 014738
@anmode done the changes

**Comment by anmode** [comment]
I have update some things....actually data api se dena. m api provide kr rha hu jo mera data fetch kr rhi h uska use krna ...
```
https://grabtern-backend.vercel.app/api/mentors/mentorDetail/anmode
```

```
{"mentorDetail":{"username":"anmode","name":"Anmol Agrawal","email":"anmol2002.aa1@gmail.com","mobile":9368086395,"internAt":"Google Summer of Code 2022","image":"https://res.cloudinary.com/grabtern-cloud/image/upload/v1688361690/og87pyq4vyptl1dlmffs.png","currentStatus":"Student software developer","description":"I have experience creating websites and web applications as a front-end developer. I have professional expertise dealing with C# and Next, and I specialize in JavaScript. I have also worked with React, typescript, and Vue. View my portfolio or get in touch!\n\nThere was more to come. I have a keen interest in blockchain-related things, ML, AI, and data science.\nSo I am also working on developing these abilities right now","social":{"linkedin":"https://www.linkedin.com/in/anmol-agrawal-748389151/","twitter":"https://twitter.com/A2_anmolagrawal","_id":"64bcf6f41cee943c2cc9a7b3"},"sessions":[{"name":"1 on 1 Mentorship","type":"video-meeting","duration":"30","description":"Achieve your goals faster with customized road map","price":"80","_id":"64bcf6f41cee943c2cc9a7b0"},{"name":"Resume Review","type":"video-meeting","duration":"15","description":"Get your resume reviewed.","price":"51","_id":"64bcf6f41cee943c2cc9a7b1"}],"schedules":[],"testimonials":[]}}
```

**Comment by coder12git** [comment]
@anmode done the changes

File: components/mentorDashboard/sessions.js [hunk 1]
```diff
@@ -1,48 +1,71 @@
...
+              return (
+                <SessionCard
+                  key={index}
+                  type={card.type}
+                  name={card.name}
+                  description={card.description}
+                  duration={card.duration}
+                  price={card.price}
+                  text="Edit Session"
+                  path="/dashboard/editMentorSession"
```
Comment (anmode) [inline components/mentorDashboard/sessions.js:63]: here we have to pass session and index: Jisse form m details aaske uss session jispr user edit krne ko click kia ... then unn details ko update kr skte h...

File: pages/dashboard/editMentorSession.page.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
+import React from "react";
```
Comment (anmode) [inline pages/dashboard/editMentorSession.page.js:1]: thoda special component h yeh as session pass hoga isss form m

**Review by anmode — APPROVED** [review]
I left comment please cover those in next pR

**Pull request merged**